components/entrySignUp/entrySignUp.js

```diff
@@ -80,11 +80,20 @@ Template.entrySignUp.helpers({
       return "border: 1px solid #a94442";
     } else if (ActiveEntry.errorMessages.equals('fullName', "Name is probably not complete")) {
       return "border: 1px solid #f2dede";
-    } else if (ActiveEntry.errorMessages.equals('fullName', "Name present")) {
+    } else if (ActiveEntry.successMessages.equals('fullName', "Name present")) {
       return "border: 1px solid green";
     } else {
       return "border: 1px solid gray";
     }
+  },
+  signUpErrorMessages: function() {
+    var errorMessages = [];
+    Object.keys(ActiveEntry.errorMessages.all()).forEach(function(key) {
+      if (key !== "signInError" && ActiveEntry.errorMessages.get(key)) {
+        errorMessages.push(ActiveEntry.errorMessages.get(key));
+      }
+    });
+    return errorMessages;
```

Comment:
I'm curious about lines 88 through 96 here.  The original implementation had an assumption that there would only be one error message ever displayed at a time (the current one that the user needs to address).  It seems you're wanting to move towards listing multiple error messages.  Which is doable, but makes things more complicated in the rest of the UI.  

Could you log an issue to discuss successMessages vs errorMessages?   I'm not necessarily against the changes; but I'd like for them to be documented and for us to have a model of what's going on.